### PR TITLE
Convert `text.module.css` to CSS modules

### LINKED ISSUE #40576: Migrate global css in `frontend/src/metabase/css/core/text.module.css`

### PR DESCRIPTION
**Pull request by oisincoveney**
### Description

Describe the overall approach and the problem being solved.

### How to verify

Describe the steps to verify that the changes are working as expected.

1. New question -> Sample Dataset -> ...
2. ...

### Demo

_Upload a demo video or before/after screenshots if sensible or remove the section_

### Checklist

- [ ] Tests have been added/updated to cover changes in this PR

### CODE REVIEW

File: frontend/src/metabase/models/components/NewModelOption.jsx [hunk 1]
```diff
@@ -42,11 +42,14 @@ export default class NewModelOption extends Component {
...
         </div>
         <div
-          className="text-normal mt2 mb2 text-paragraph"
+          className={cx(CS.textNormal, "mt2 mb2", CS.textParagraph)}
           style={{ lineHeight: "1.25em" }}
         >
           <h2 className={cx(S.modelTitle, { "text-brand": hover })}>{title}</h2>
-          <p className="text-medium text-small" style={{ maxWidth: "360px" }}>
+          <p
+            className={cx("text-medium", CS.textSmall)}
```
Comment (npretto) [inline frontend/src/metabase/models/components/NewModelOption.jsx:50]: text-medium is a color, text-small is a size 😭

Comment (oisincoveney) [reply frontend/src/metabase/models/components/NewModelOption.jsx:50]: oh yes! 😂

**Review by npretto — APPROVED** [review]

**Review by WiNloSt — COMMENTED** [review]
77/110 files reviewed. I'll continue after lunch.

File: frontend/src/metabase/query_builder/components/AlertModals/AlertModals.jsx [hunk 2]
```diff
@@ -196,15 +196,17 @@ export class AlertEducationalScreen extends Component {
     const { onProceed } = this.props;
 
     return (
-      <div className="pt2 pb4 ml-auto mr-auto text-centered">
+      <div className={cx("pt2", "pb4", "ml-auto", "mr-auto", CS.textCentered)}>
```
[the remark this answers is not in the document]
Comment (oisincoveney) [reply frontend/src/metabase/query_builder/components/AlertModals/AlertModals.jsx:199]: Yeah, I thought about that too late. The migration is already done in other PRs already so I'll deal with the merge conflicts

File: frontend/src/metabase/components/TokenField/TokenField.tsx [hunk 3]
```diff
@@ -648,7 +648,16 @@ class _TokenField extends Component<TokenFieldProps, TokenFieldState> {
             <li className="mr1" key={this._key(option)}>
               <div
                 className={cx(
-                  `py1 pl1 pr2 block rounded text-bold text-${color}-hover inline-block full cursor-pointer`,
+                  CS.py1,
```
[the remark this answers is not in the document]
Comment (oisincoveney) [reply frontend/src/metabase/components/TokenField/TokenField.tsx:651]: Yep, makes sense. I'll revert the changes here

**Review by WiNloSt — APPROVED** [review]
awesome

**Pull request merged**